rustfmt.toml

```diff
@@ -64,7 +64,7 @@ use_field_init_shorthand = false
 force_explicit_abi = true
 condense_wildcard_suffixes = false
 color = "Auto"
-required_version = "1.5.1"
+required_version = "1.5.2"
```

Comment:
1.5.1 is the latest release

Reply:
It kept complaining for me that my version is 1.5.2. If you have Rust installed via rustup try `rustup update` it doesn't update automatically. If you haven't installed it via rustup it is recommended fir development and this file is completely irrelevant to the end user

Reply:
Weird, the latest release on their GitHub is 1.5.1. I used the Fedora packaged rust version (1.68.2) so I'll use rustup instead :)